src/routes/authRoutes.js

```diff
@@ -1,9 +1,8 @@
-import { Router } from "express";
-import passport from "passport";
-import Jwt from "../utils/jwt.js";
-import { googlePass } from "../utils/passport.js";
+import { Router } from 'express';
+import passport from 'passport';
+import Jwt from '../utils/jwt.js';
+import { googlePass } from '../utils/passport.js';
 import profileController from '../controller/profileController.js';
-const router = Router();
 
```

Comment:
can you add this line of code, so that you can be able to use Vendor, as you know we lost some lines of codes, after rebase of some branches
**import vendor from '../controller/vendorController.js';**

Reply:
Done